Mathlib/Topology/Order/ScottTopology.lean

```diff
@@ -0,0 +1,364 @@
+/-
+Copyright (c) 2023 Christopher Hoskin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Christopher Hoskin
+-/
+
+import Mathlib.Topology.Order.UpperLowerSetTopology
+
+/-!
+# Scott topology
+
+This file introduces the Scott topology on a preorder.
+
+## Main definitions
+
+- `DirSupInacc` - a set `u` is said to be inaccessible by directed joins if, when the least upper
+  bound of a directed set `d` lies in `u` then `d` has non-empty intersection with `u`.
+- `Scott` - the Scott topology is defined as the join of the topology of upper sets and the
```

Comment:
Can you check all the names in the docstring and make sure they're up to date and all fully qualified?

For example, this should be
```suggestion
- `Topology.scott` - the Scott topology is defined as the join of the topology of upper sets and the
```

Reply:
I think this is done now.